cmake/Templates/opm-project-config.cmake.in

```diff
@@ -104,6 +114,7 @@ if(NOT @opm-project_NAME@_FOUND)
     @OPM_PROJECT_EXTRA_CODE@
     # end extra code
 
+    include(OpmPackage)
```

Comment:
only thing that scares me a little is the sibling search done inside here by default.

Reply:
I do not see a way to do that unless we remove all build system magic for finding and use plain modern cmake. It is used everywhere.

Note that this does not change anything  within OPM, but it fixes calling find_package(opm-common) in any external plain CMake project.

Reply:
Yeah, I know what you want to achieve and that this is nothing new. 

Originally I wanted to use standard cmake stuff for depends and not look for depends of depends (import targets, or old style xxx_YYY variables) but somehow that broke down, and we (you actually) were forced to re-add the finding of deps in the package req file. I think we should chase the original goal later, just wanted to point it out.